examples/howto/data_container.myst.md

```diff
@@ -24,26 +24,34 @@ import arviz as az
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
-import pymc3 as pm
+import pymc as pm
 
 from numpy.random import default_rng
 
-print(f"Running on PyMC3 v{pm.__version__}")
+print(f"Running on PyMC v{pm.__version__}")
 ```
 
 ```{code-cell} ipython3
 %matplotlib inline
 %config InlineBackend.figure_format = 'retina'
-RANDOM_SEED = 8927
+RANDOM_SEED = sum(map(ord, "Data Containers in PyMC5"))
 rng = default_rng(RANDOM_SEED)
 az.style.use("arviz-darkgrid")
 ```
 
-## The Data class
+## Data Containers
 
-The {class}`pymc.Data` container class wraps the theano shared variable class and lets the model be aware of its inputs and outputs. This allows one to change the value of an observed variable to predict or refit on new data. All variables of this class must be declared inside a model context and specify a name for them.
+The {class}`pymc.Data` container allows users to keep track of dimensions (like dates or cities) and coordinates (such as the actual date times or city names) of multi-dimensional data. It offers two wrappers for this: {class}`pymc.ConstantData` and {class}`MutableData`. Both allow you to specify the dimension names and coordinates of random variables, instead of specifying the shapes of those random variables as numbers. The difference is whether you intend to change this data after fitting the model, to perform out-of-sample prediciton, for example. 
 
-In the following example, this is demonstrated with fictional temperature observations.
+In past versions of PyMC, the only data container was `pm.Data`. This container is still available for backwards compatability, but the current best practice is to use either `pm.MutableData` or `pm.ConstantData`, depending on your needs.  This notebook will demonstrate how each of these containers are used, and when they are appropriate.
```

Comment:
I inferred it from [this warning](https://github.com/pymc-devs/pymc/blob/6bfbff508b80e588cc03711798b4e2577ad46449/pymc/data.py#L432). If you think I'm making the case too strongly I can soften this to something like "you can set `mutable=True/False` in `pm.Data` or be explicit by using `pm.MutableData`/`pm.ConstantData`."

That said in my opinion the recommended best practice should be to use the specific containers instead of the mutable flag,